pyccel/codegen/printing/fcode.py

```diff
@@ -481,7 +465,8 @@ def _print_Print(self, expr):
 
     def _print_SymbolicPrint(self, expr):
         # for every expression we will generate a print
-        _iprint = lambda e: "print *, 'sympy> {}'".format(e)
+        def _iprint(e):
+            return "print *, 'sympy> {}'".format(e)
```

Comment:
Is there a reason for removing the lambda function?

Reply:
I think I removed this as it gave a codacy warning:
`W0108: Lambda may not be necessary (unnecessary-lambda)`
Although it may have meant that we can use this expression without defining a function (it is only used once) rather than that it would be better to use a function rather than a lambda function
```
Name: Unnecessary lambda

Description: A lambda is used that calls through to a function without modifying any parameters
```